lib/DirectAdmin_API.php

```diff
@@ -47,14 +47,32 @@ public function __construct($host = null, $login = null, $pass = null, $secure =
 			$this->_setupLocalConnection();
 	}
 	
-	public function getDomainsMxRecords($domain){
+	public function getDomainsMxRecords($domain, $username){
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->